src/components/UI/dropdown/dropdown.module.scss

```diff
@@ -0,0 +1,136 @@
+.userGreet {
+  color: #ffffff;
+  margin: 23px;
+}
+
+.userGreet a {
+  padding: 0px;
+}
+
+.userProfilePic {
+  width: 32px;
+  height: 32px;
+  border-radius: 50%;
+  display: inline;
+  vertical-align: middle;
+}
+.dropdownHeader {
+  margin: 10px;
+  cursor: pointer;
+}
+
+.userGreetMsg {
+  display: inline;
+  vertical-align: middle;
+  margin-right: 10px;
+  color: rgb(255, 255, 255);
+}
+
+.userGreetMsg:hover {
+  color: #49a82e;
+}
+
+.myProfileWrapper {
+  padding-bottom: 15px;
+  text-align: center;
+  border-bottom: 1px solid #000000;
+}
+
+.myProfile:hover {
+  color: #49a82e;
+}
```

Comment:
can we put these colors in css variables please?

Reply:
made the changes as you suggested please have look.